### ISSUE TITLE
Spider charge targets need consistent mapping

### ISSUE THREAD

**Issue by slarticodefast** [space-wizards/space-station-14#32864]
## Description
The possible targets for the spider charge are selected from warp points with the BombingTargetComponent attached:
![grafik](https://github.com/user-attachments/assets/43767ddc-8b5f-48e3-8518-5d4264077d5b)

There is a specific warp point prototype for mappers to use:
![grafik](https://github.com/user-attachments/assets/dfd1d46f-afd7-458d-ae31-965528759be6)

However, only Core and Saltern actually use this:
![grafik](https://github.com/user-attachments/assets/cf67118f-d5b8-4617-b839-41a9ab2c6111)

And Train attaches the component manually to the warp points instead of using the prototype:
![grafik](https://github.com/user-attachments/assets/20a9e85d-3804-4589-810d-4546511055c4)

**Reproduction**
- Load a map other than Core, Saltern or Train
- Spawn a ninjas and take the ghost role
- never get the spider charge objective

**Additional context**
To resolve this either map the WarpPointBombing prototype correctly or make the objective use something else to select the bombing target.

The spider charge is supposed to only explode near a valid target. However, if the objective is not correctly assigned, the ninja still gets it in his loadout and it can be placed anywhere.

**Comment by JIPDawg** [comment]
Just to clarify what was said in #cartography I think warp points are kill and everyone should be using beacons now as they double as warp points.

**Comment by slarticodefast** [comment]
Yeah, adding `BombingTargetComponent` to a beacon should work as well

**Comment by Errant-4** [comment]
> Yeah, adding `BombingTargetComponent` to a beacon should work as well

This means theoretically a ninja could just unwrench their target beacon and place it where they want to bomb

**Comment by deltanedas** [comment]
or just remove the funny thing and let them bomb anything

**Comment by JIPDawg** [comment]
> or just remove the funny thing and let them bomb anything

Goodbye arrivals, Hello Tesloose :trollface:

**Comment by Errant-4** [comment]
> > or just remove the funny thing and let them bomb anything
> 
> Goodbye arrivals, Hello Tesloose :trollface:

Ninjas don't need a bomb to loose the engine, though, do they? They can just emag the PA computer

**Comment by baddiePRO** [comment]
> or just remove the funny thing and let them bomb anything

this please, I need my "AI bolted and depowered the server room" bomb

**Comment by baddiePRO** [comment]
> > > or just remove the funny thing and let them bomb anything
> > 
> > Goodbye arrivals, Hello Tesloose :trollface:
> 
> Ninjas don't need a bomb to loose the engine, though, do they? They can just emag the PA computer

ninja gloves can only doorjack and do the ninja objectives, you can snip the limiter wire though

**Comment by deltanedas** [comment]
or just teleport in, wall up the emitters and or snip mv, teleport out

**Comment by baddiePRO** [comment]
if it's singulo you'd be cooked before you can do that, ninja has no rad resistance
